main.py

```diff
@@ -1,22 +1,66 @@
-import win32com.client
+from dataclasses import dataclass
+import win32com.client as win32
+import sys
+
+if len(sys.argv) < 2:
+    print("Should be called with the name of the aspen file")
+    exit(1)
 
 # Connect to Aspen Plus
-aspen = win32com.client.Dispatch("Apwn.Document")
-aspen.InitFromFile(r"C:\Programming\delft\Aspen\cstr-ch4.apw")
-aspen.Visible = True
+aspen = win32.gencache.EnsureDispatch("Apwn.Document")
+aspen.InitFromArchive2(sys.argv[1])

```

Comment:
Does init from archive2 do something else then init from file? Does it expect some different file type